spec/lib/ruby_smb/dcerpc/winreg/enum_key_response_spec.rb.orig

```diff
@@ -0,0 +1,63 @@
+RSpec.describe RubySMB::Dcerpc::Winreg::EnumKeyResponse do
```

Comment:
This .orig file looks like it was accidentally committed while editing the git history?

Reply:
Yes, my bad :/